CinemaTicketSalesSystem/CinemaTicketSalesSystem/App_Start/AutofacConfig.cs

```diff
@@ -18,13 +23,47 @@ public static IContainer ConfigureContainer()
             builder.RegisterControllers(typeof(MvcApplication).Assembly);
             builder.RegisterType<ApplicationDbContext>().AsSelf().InstancePerRequest();
             builder.Register<UserStore<ApplicationUser>>(c => new UserStore<ApplicationUser>(c.Resolve<ApplicationDbContext>())).AsImplementedInterfaces();
-            //builder.RegisterGeneric(typeof(UserStore<>)).As(typeof(IUserStore<>)).InstancePerRequest();
             builder.Register<IdentityFactoryOptions<ApplicationDbContext>>(c => new IdentityFactoryOptions<ApplicationDbContext>()
             {
                 DataProtectionProvider = new Microsoft.Owin.Security.DataProtection.DpapiDataProtectionProvider("CinemaTicketSalesSystem")
             }).InstancePerRequest();
             builder.RegisterType<ApplicationUserManager>().AsSelf().InstancePerRequest();
-            builder.RegisterType<DbService>().As<IDbService>().WithParameter("context", new ApplicationDbContext());
+            builder.RegisterType<MovieService>().As<IMovieService>().InstancePerRequest();
+            builder.RegisterType<ActorService>().As<IActorService>().InstancePerRequest();
+            builder.RegisterType<ProducerService>().As<IProducerService>().InstancePerRequest();
+            builder.RegisterType<GenreService>().As<IGenreService>().InstancePerRequest();
+
+
+            builder.Register(context =>
+            {
+                var config = new MapperConfiguration(x =>
+                {
+                    x.AddProfiles(typeof(MovieProfile).Assembly);
+                    x.AddProfiles(typeof(PictureProfile).Assembly);
```

Comment:
you don't need to add each assembly from each Profile.
they are all already in one assembly.
You need to add here just one line
x.AddProfiles(typeof(MovieProfile).Assembly);
and it will work for others